src/helpers/workflow/workflow-helper.js

```diff
@@ -12,7 +12,15 @@ export async function fetchWorkflowsWithGroups({ limit = 10, offset = 0 }) {
   let wfData = await workflowApi.listWorkflows(limit, offset);
   let workflows = wfData.data;
   return Promise.all(workflows.map(async wf => {
-    let wfWithGroups = await fetchGroupNames(wf.group_refs);
+    let wfWithGroups = wf.group_refs;
+    try {
+      wfWithGroups = await fetchGroupNames(wf.group_refs);
+    }
+    catch (error) {
```

Comment:
I would probably not use try catch block but use the `.catch` method on the `PromiseAll` object (same as then). Because it is a `Promise` it will end up there anyway.

If you do that then you can add the await hook to the return statement and return correct promise object to then event chain:
```javascript
...
return { ...wf, group_names: await: fetchGroupNames(wf.group_refs }
```